### PR TITLE
Task/barcode icon

### PR DESCRIPTION
**Pull request by Andreolle**
#### Short description of what this resolves:
Adding **barcode.svg** to _sling-assets_ and registering this icon in _sling-web-component-icon_

### Testing
`npm run start sling-web-component-icon`

#### Changes proposed in this pull request:

-
-
-

### CODE REVIEW

File: packages/sling-assets/src/icons/js/boleto.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          <rect x="439.966" y="54.816" width="26.837" height="401.707"/>
+          <rect x="475.653" y="54.816" width="9.134" height="401.707"/>
+          <rect x="493.64" y="54.816" width="17.986" height="401.994"/>
+          <rect x="332.045" y="54.816" width="17.987" height="401.707"/>
+          <rect x="368.019" y="54.816" width="17.987" height="401.707"/>
+          <rect x="296.072" y="54.816" width="17.986" height="401.707"/>
+          <rect x="251.243" y="54.816" width="17.989" height="401.707"/>
+      </g>
+  </g>
+  <g></g><g></g>
```
Comment (leofavre) [inline packages/sling-assets/src/icons/js/boleto.js]: All empty `<g></g>` should be removed.

Comment (Andreolle) [reply packages/sling-assets/src/icons/js/boleto.js]: done! :smiley:

File: packages/sling-web-component-brand-icon/src/component/BrandIcon.js [hunk 2]
```diff
@@ -79,6 +79,8 @@ export class BrandIcon extends HTMLElement {
         return icons.HIPERCARD;
       case 12:
         return icons.SENFF;
+      case 17:
+        return icons.BOLETO;
```
Comment (Luan-BK) [inline packages/sling-web-component-brand-icon/src/component/BrandIcon.js]: Esse ID é na verdade de outra bandeira, tem que ver outra forma de fazer esse mapeamento para boleto.

Comment (Andreolle) [reply packages/sling-web-component-brand-icon/src/component/BrandIcon.js]: como conversamos adicionei ele em sling-icon

File: packages/sling-assets/src/icons/index.js [hunk 3]
```diff
@@ -15,3 +15,4 @@ export { SENFF } from './js/cardbrand-senff.js';
 export { ELO } from './js/cardbrand-elo.js';
 export { STONE } from './js/cardbrand-stone.js';
 export { ELLIPSIS } from './js/ellipsis.js';
+export { BOLETO } from './js/boleto.js';
```
Comment (biancarosa) [inline packages/sling-assets/src/icons/index.js]: It could be bankslip here....

Comment (breakzplatform) [reply packages/sling-assets/src/icons/index.js]: I dont think so. Boleto is something exclusive in the world. Some companies, like Ebanx, Paypal and GearBest uses "Boleto" even on english translations.

**Pull request merged**